layouts/_default/baseof.html

```diff
@@ -24,93 +65,42 @@
-
-<!-- GC Design System -->
-<link rel="stylesheet" href="https://cdn.design-system.alpha.canada.ca/@cdssnc/gcds-components@0.22.0/dist/gcds/gcds.css">
-<script type="module" src="https://cdn.design-system.alpha.canada.ca/@cdssnc/gcds-components@0.22.0/dist/gcds/gcds.esm.js"></script>
-<script nomodule src="https://cdn.design-system.alpha.canada.ca/@cdssnc/gcds-components@0.22.0/dist/gcds/gcds.js"></script>
-{{ if (eq .RelPermalink "/")}}
-<meta name="description" content="{{ i18n "meta-description" }}"/>
-<meta name="keywords" content="{{ i18n "meta-keywords" }}"/>
-<meta name="dcterms.title" content="{{ i18n "meta-title" }}"/>
-<meta name="dcterms.description" content="{{ i18n "meta-description" }}"/>
-<meta name="dcterms.creator" content="{{ i18n "canadian-digital-service" }}"/>
-<meta name="dcterms.language" title="ISO639-2/T" content="{{ i18n "aa-language" }}"/>
-<meta name="dcterms.subject" title="gccore" content="{{ i18n "meta-subject" }}"/>
-<meta name="dcterms.issued" title="W3CDTF" content="2013-12-13"/>
-<meta name="dcterms.modified" title="W3CDTF" content="2024-04-16"/>
-<meta name="dcterms.identifier" content="{{ i18n "meta-identifier" }}"/> 
-{{ end }}
+  <!-- Font Awesome (Icons) -->
+  <!-- <script src="https://kit.fontawesome.com/892cb27850.js" crossorigin="anonymous"></script> -->
 </head>
 
 {{ with $pageTitle }}
-
-<body vocab="http://schema.org/" typeof="WebPage" class="{{lower $titleKey | urlize}}">
-  {{ else }}
-
-  <body vocab="http://schema.org/" typeof="WebPage"
-    class="{{if eq .Section "blog"}}page-blogpost{{else}}{{lower  $titleKey | urlize}}{{end}}">
-    {{end}}
+<body vocab="http://schema.org/" typeof="WebPage" class="{{ lower $titleKey | urlize }}">
+{{ else }}
+<body vocab="http://schema.org/" typeof="WebPage" class='{{ if eq .Section "blog"}}page-blogpost{{ else }}{{ lower  $titleKey | urlize }}{{ end }}'>
+{{ end }}
 
     <!-- Google Tag Manager DO NOT REMOVE OR MODIFY - NE PAS SUPPRIMER OU MODIFIER -->
     <!-- EN -->
-    <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-PZ6LLXT" height="0" width="0"
-        style="display:none;visibility:hidden"></iframe></noscript>
-
+    <noscript>
+      <iframe src="https://www.googletagmanager.com/ns.html?id=GTM-PZ6LLXT" height="0" width="0" style="display:none;visibility:hidden"></iframe>
+    </noscript>
     <!-- FR -->
-    <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-T7JHWPJ"
-      height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
+    <noscript>
+      <iframe src="https://www.googletagmanager.com/ns.html?id=GTM-T7JHWPJ" height="0" width="0" style="display:none;visibility:hidden"></iframe>
+    </noscript>
     <!-- End Google Tag Manager -->
 
-    <nav aria-label="{{ i18n "skip-links" }}">
+    <nav aria-label='{{ i18n "skip-links" }}'>
```

Comment:
Just curious about the switch to single quotes here for attributes.  Does the original code with double quotes break because of the nested double quotes?

Reply:
It doesn't break, it just looks like an error in some IDEs (like mine).